src/plugins/external_tool_support/src/python/PythonSupport.cpp

```diff
@@ -34,6 +34,12 @@
 
 namespace U2 {
 
+namespace {
+
+const QString PYTHON_VERSION_REGEXP = "(\\d+.\\d+.\\d+)";
```

Comment:
6 lines of code for a constant that can be a local variable. Since this constant is used inside of the 1 method you can move it into that method or you can use static/#define to make it 1-liner.

Reply:
`#define` for constants is bad.

Reply:
#define is a tool and can't be good or bad. Its usages can be good or bad. 

There is nothing bad when you use #define for local constants for primitive types.